src/DynamoCoreWpf/ViewModels/Core/WorkspaceViewModel.cs

```diff
@@ -467,6 +468,16 @@ public WorkspaceViewModel(WorkspaceModel model, DynamoViewModel dynamoViewModel)
             InCanvasSearchViewModel = new SearchViewModel(DynamoViewModel);
             InCanvasSearchViewModel.Visible = true;
         }
+        /// <summary>
+        /// This event is triggred from Workspace Model. Used in instrumentation
+        /// </summary>
+        /// <param name="modelData"> Workspace model data as JSON </param>
+        /// <returns>workspace model with view block in string format</returns>
+        private string Model_PopulateJSONWorkspace(string modelData)
```

Comment:
It feels really weird to me that this just accepts any string... considering it will throw in `AddViewBlockToJSON` given some other string... couldn't we use a JObject instead or at least document what the string should be like.

Reply:
AddViewBlockToJSON is a private function. I accept this if that function is internal or protected or public. I think using string provides a clear representation than a jobject.